Sources/PluginLibrary/Descriptor.swift

```diff
@@ -56,35 +57,70 @@ public final class DescriptorSet {
 }
 
 public final class FileDescriptor {
+  public enum Syntax: String {
+    case proto2
+    case proto3
+
+    public init?(rawValue: String) {
+      switch rawValue {
+      case "proto2", "":
+        self = .proto2
+      case "proto3":
+        self = .proto3
+      default:
+        return nil
+      }
+    }
+  }
+
   public let proto: Google_Protobuf_FileDescriptorProto
-  public var name: String { return self.proto.name }
+  public var name: String { return proto.name }
+  public var package: String { return proto.package }
+
+  public let syntax: Syntax
+
+  public var dependencies: [String] { return proto.dependency }
+  public var publicDependencies: [String] { return proto.publicDependency.map { dependencies[Int($0)] } }
```

Comment:
Would these benefit from being `lazy` so that the map is only computed once when it's first requested?